routes/users.route.ts

```diff
@@ -37,6 +45,35 @@ function getArtistsAmount(req: Request, res: Response): void {
     });
 }
 
+function getUserNotifications(req: Request, res: Response): void | undefined {
+  const query = req.query;
+
+  if (!query.userId) {
+    const status = 500;
+    res.status(status).send({message: 'Unidentified user'});
```

Comment:
maybe it'll be good to make a separate const with message options?

Reply:
I'll do it later. I 'll create new PR where I fix it on all project